conf/base.py

```diff
@@ -22,6 +22,10 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Versions for engine subscription
+LOAD_VERSION = os.environ.get('LOAD_VERSION', '1.1.0')
```

Comment:
These don't need to be environment variables. The version specified should be constant for a given version of the transmission codebase.